server/plugin/webhook.go

```diff
@@ -1377,3 +1401,9 @@ func (p *Plugin) postStarEvent(event *github.StarEvent) {
 		}
 	}
 }
+
+func (p *Plugin) logForDocsRepoDebugging(repo, message string, kvMessage ...interface{}) {
+	if repo == "mattermost/docs" {
```

Comment:
@hanzei Since this is a build just for debugging the issue. I don't think there is a need to add this logic here. Are there any specific risks you are thinking of here?

Reply:
If we want to have this permanently in the plugin, I think @hanzei's suggestion makes sense. Note that there could be an issue with configuring the subscription itself though, so this may not help us in all cases. I do think it would help us debug this particular issue though.

Also would we want to support this for all event types?

@hanzei I'm wanting to get something on community as soon as we can to help debug the issue we're experiencing, so I don't want to prolong the PR for the goal of supporting a general use case. Though I see the value in it, and 1/5 if we can get it merged efficiently then I'm fine with this.

Reply:
@mickmister @hanzei Maybe we can create a separate Github issue for this and use the current build for debugging?